comparative_benchmark/jax_xla/benchmark_all.sh

```diff
@@ -72,6 +72,19 @@ fi
 
 "${TD}/../scripts/create_results_json.sh" "${OUTPUT_PATH}"
 
+VENV_DIR_PATH="$(realpath ${VENV_DIR})"
+PYTHON_VERSION="$(python --version | sed -e "s/^Python \(.*\)\.\(.*\)\..*$/\1\.\2/g")"
+
+# `transformers` is not yet up to date with JAX so we need to hack the source to make it compatible.
+BERT_SOURCE_PATH="${VENV_DIR_PATH}/lib/python${PYTHON_VERSION}/site-packages/transformers/models/bert/modeling_flax_bert.py"
+sed -i -e "s/: Optional\[jnp.DeviceArray\] = None/=None/g" "${BERT_SOURCE_PATH}"
+
+T5_SOURCE_PATH="${VENV_DIR_PATH}/lib/python${PYTHON_VERSION}/site-packages/transformers/models/t5/modeling_flax_t5.py"
+sed -i -e "s/: Optional\[jnp.DeviceArray\] = None/=None/g" "${T5_SOURCE_PATH}"
+
+GPT2_SOURCE_PATH="${VENV_DIR_PATH}/lib/python${PYTHON_VERSION}/site-packages/transformers/models/gpt2/modeling_flax_gpt2.py"
+sed -i -e "s/: Optional\[jnp.DeviceArray\] = None/=None/g" "${GPT2_SOURCE_PATH}"
+
```

Comment:
How often does the transfomers library release? I'm trying to balance between a temporary patch and wait for a little bit long.

Alternatively, we can install the transformers from the main branch (maybe pinned at a commit). They seem to give out an official instruction to do that (https://huggingface.co/docs/transformers/installation#install-from-source).

Reply:
It looks like they release every month and the last one happened ~3 weeks ago. We should expect one in the next week. Happy to wait or merge this and remove once it's released.

Reply:
Yeah, I prefer to wait for the release assuming it will come out next week. In the meantime we can tune the timeout problem on this PR

Reply:
Seems like they haven't released a new version yet. Feel free to file an issue and merge this

Reply:
Sounds good. I'll keep an eye out for a new HuggingFace release.